functests/performance/updating_profile.go

```diff
@@ -58,10 +62,9 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 
 		// Modify profile and verify that MCO successfully updated the node
 		testutils.BeforeAll(func() {
-			// timeout should be based on the number of worker nodes
-			timeout := time.Duration(len(workerRTNodes) * mcpUpdateTimeout)
-
 			By("Modifying profile")
```

Comment:
Let's move this By below initialProfile.

Reply:
not really agree.. as for me - it's more readable: save initialProfile before making changes